scss/styles.scss

```diff
@@ -1,3 +1,4 @@
 .title {
   color: red;
 }
+
```

Comment:
Is this file being used? If not it can be removed so we don't have more files than we need in the repo